documentation/principles/native-first.md

```diff
@@ -8,5 +8,6 @@ tags:
 eleventyNavigation:
   parent: principles
 ---
```

Comment:
How do we build this out? 
OR does it get folded into security?
Or cost effective?